### PR TITLE
FIxed bug with document (#1927)

### PR DESCRIPTION
**Pull request by DmytroZelinsky**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by VSBalanchuk — APPROVED** [review]

File: src/pages/GoverningBody/AddDocumentModal/AddDocumentModal.tsx [hunk 1]
```diff
@@ -69,13 +77,17 @@ const AddDocumentModal = (props: Props) => {
         setDisabled(true);
       }
       
+      const isEmptyFile = fileSize !== 0;
+      if (!isEmptyFile)
```
Comment (VolodymyrKostiv) [inline src/pages/GoverningBody/AddDocumentModal/AddDocumentModal.tsx:81]: Why this variable is called 'isEmptyFile'? If 'fileSize' is bigger than 0, than the result of expression is true, which means that your file is not empty. So, change the variable name or rewrite the condition. Also, in this if statement it looks like you're checking if file is not empty, and if it's true, you're putting the notification that it's empty)
Also, please, specify the variable type.

File: src/pages/GoverningBody/Sector/AddDocumentModal.tsx [hunk 2]
```diff
@@ -76,13 +77,17 @@ const AddDocumentModal = (props: Props) => {
       setDisabled(true);
     }
 
+    const isEmptyFile = fileSize !== 0;
+    if (!isEmptyFile)
```
Comment (VolodymyrKostiv) [inline src/pages/GoverningBody/Sector/AddDocumentModal.tsx:81]: Same situation with variable name

File: src/pages/RegionsBoard/AddDocModal.tsx [hunk 3]
```diff
@@ -105,6 +106,11 @@ const AddDocumentModal = (props: Props) => {
       extension.indexOf("pdf") !== -1 ||
       extension.indexOf("doc") !== -1 ||
       extension.indexOf("docx") !== -1;
+      
+    const isEmptyFile = fileSize !== 0;
+    if (!isEmptyFile)
```
Comment (VolodymyrKostiv) [inline src/pages/RegionsBoard/AddDocModal.tsx:111]: Same situation with variable name

File: src/pages/GoverningBody/AddDocumentModal/AddDocumentModal.tsx [hunk 4]
```diff
@@ -69,13 +77,17 @@ const AddDocumentModal = (props: Props) => {
...
+      notificationLogic("error", fileIsEmpty());
+
       const isSmaller3mb = fileSize < 3145728;
```
Comment (VolodymyrKostiv) [inline src/pages/GoverningBody/AddDocumentModal/AddDocumentModal.tsx:84]: I know that this is not your code, but please, remove this magic number and use constant instead

**Comment by DmytroZelinsky** [comment]
> Resolve Volodymyr's comments 💩

ok 💩

**Pull request merged**